app/controllers/hyrax/articles_controller.rb

```diff
@@ -7,6 +7,8 @@ class ArticlesController < ApplicationController
     # Adds Hyrax behaviors to the controller.
     include Hyrax::WorksControllerBehavior
     include Hyrax::BreadcrumbsForWorks
+    include HykuAddons::WorksControllerAdditionalMimeTypesBehavior
```

Comment:
Do you guys think we could move this line to `Hyrax::WorksControllerBehavior` to avoid repeating it into every works controller?

Reply:
I think I avoided this initially because of the autoloading. But yes, this is a better solution.

Reply:
However, I'm not sure it works adding it to the engine like: 

`Hyrax::WorksControllerBehavior.include HykuAddons::WorksControllerAdditionalMimeTypesBehavior`

I'm now getting Unknown Format errors.

Reply:
Oh, I forgot that `Hyrax::WorksControllerBehavior` isn't something that we're supplying.  How about we make a `HykuAddons::WorksControllerBehavior` where we can gather all of our includes and include that in each controller?  That way it is explicit and we don't have to inject it in an initializer but avoids having to add another include when the next mixin comes along.

Reply:
@cjcolvar Are you referring to the [Hyrax::My::WorksController](https://github.com/samvera/hyrax/blob/master/app/controllers/hyrax/my/works_controller.rb)? 

I'm not sure where to include this behaviour override.